cmake/features.cmake

```diff
@@ -44,6 +44,8 @@ endif()
 
 ov_dependent_option (ENABLE_ONEDNN_FOR_GPU "Enable oneDNN with GPU support" ${ENABLE_ONEDNN_FOR_GPU_DEFAULT} "ENABLE_INTEL_GPU" OFF)
 
+ov_option (ENABLE_INTEL_NPU "NPU plugin for OpenVINO runtime" OFF)
```

Comment:
why plugin is OFF?

Reply:
you can override some flags with `set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=undef")`

Reply:
Removed -Werror for linux, /WX for windows, and add special flags for level zero repo to pass build. We meet vc140.pdb access issue in job, so use /Z7 for it. The pdb issue shall relate to OV CI machine, FYI